jung-algorithms/src/main/java/edu/uci/ics/jung/algorithms/layout/util/VisRunner.java

```diff
@@ -10,6 +10,7 @@
 package edu.uci.ics.jung.algorithms.layout.util;
 
 import edu.uci.ics.jung.algorithms.util.IterativeContext;
+import org.apache.log4j.Logger;
```

Comment:
Please remove the logging code (here and elsewhere)--both the Logger stuff and the System.out.println()s.